### ISSUE TITLE
Can you please add Musl to cross compile? 

### ISSUE THREAD

**Issue by jasonjurotich** [vapor/postgres-nio#491]
Apple is saying here that they are going to use Musl for cross compiling... can you add it please? Thanks 
https://www.swift.org/documentation/articles/static-linux-getting-started.html

#if os(macOS) || os(iOS)
import Darwin
#elseif canImport(Glibc)
import Glibc
#elseif canImport(Musl)
import Musl
#elseif os(Windows)
import ucrt
#else
#error(Unknown platform)
#endif

**Comment by MihaelIsaev** [comment]
It seems there are only three places where we `import Glibc`, and only two of them need changes?
<img width="523" alt="Screenshot 2024-07-02 at 02 17 49" src="https://github.com/vapor/postgres-nio/assets/1272610/5bd7b901-907f-4fe2-904e-007e4e9ff8cb">

**Comment by jasonjurotich** [comment]
yes, but principally here: postgres-nio/Sources/ConnectionPoolModule/PoolStateMachine.swift:4:8: error: no such module 'Glibc'

**Comment by jasonjurotich** [comment]
hello, just wanted to know if this was added yet... I had to clone this weeks ago to add it myself while waiting. Thanks!

**Comment by thoven87** [comment]
> hello, just wanted to know if this was added yet... I had to clone this weeks ago to add it myself while waiting. Thanks!

Why not open a Pull request with your patch?

**Comment by ptoffy** [comment]
This was resolved in https://github.com/vapor/postgres-nio/pull/510
